src/test/java/io/fabric8/maven/docker/util/VolumeBindingUtilTest.java

```diff
@@ -302,4 +302,18 @@ public void testIsRelativePathForWindows() {
         assertFalse(isRelativePath("C:\\"));                               // C:\
         assertFalse(isRelativePath("\\"));                                 // \
     }
+
+    /**
+     * Insures that a host volume binding string that contains a windows path with .. is correctly canonicalized
+     */
+    @Test
+    public void testResolveAbsoluteWindowsVolumePath() {
+        String volumeString = format(BIND_STRING_FMT, "C:\\dir/subdir/../", CONTAINER_PATH);
```

Comment:
your test is passing on windows but not on Linux. I think we should refactor this to pick different volume strings depending upon underlying OS

Reply:
I wonder whether there is annotation for JUnit to run tests on specific platforms only. If this is possible we can just create two tests, one for windows and one for unix like operating systems.

Reply:
As far as I know the annotation for running Test on a specific platform (@EnabledOnOs) is Junit 5 specific. I added 
`Assume.assumeTrue(System.getProperty("os.name").toLowerCase().startsWith("win")); `
in the test code

Reply:
I think for now this is okay too. Let's try to get this merged. We can refactor this later in case we migrate to Junit5 in future